dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -202,7 +202,6 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 		"integration/kubernetes/alluxio-journal-volume.yaml.template",
 		"integration/kubernetes/alluxio-master.yaml.template",
 		"integration/kubernetes/alluxio-worker.yaml.template",
-		"integration/kubernetes/conf/alluxio.properties.template",
```

Comment:
can you please add `integration/kubernetes/alluxio-configMap.yaml.template` instead? thx

Reply:
Done, thanks!